File: PATTERNS/Pattern1/Pattern1.c
#include <stdio.h>

int main()
{
    int rows;
    
    printf("Enter the number of rows: ");
    scanf("%d", &rows);
    
    // Print upper half of the pyramid
    for (int i = 0; i < rows; i++)
    {
        // Print spaces
        for (int j = 0; j < i; j++)
        {
            printf(" ");
        }

        // Print stars
        for (int j = 0; j < rows - i; j++)
        {
            printf("* ");
        }

        printf("\n");
    }
    
    // Print lower half of the pyramid
    for (int i = 0; i < rows; i++)
    {
        // Print spaces
        for (int j = 0; j < rows - i - 1; j++)
        {
            printf(" ");
        }
        
        // Print stars
        for (int j = 0; j <= i; j++)
        {
            printf("* ");
        }
        
        printf("\n");
    }

    return 0;
}

File: PATTERNS/Pattern1/Pattern1.py
n = int(input("Enter the number of rows:"))

#Printing the top half of the pattern

for i in range(0,n):
    print(" " * i, sep = "", end="")
    print("* " * (n-i), sep = "")


#Printing the bottom half of the pattern

for i in range(1,n+1):
    print(" " * (n-i), sep = "", end="")
    print("* " * i , sep = "")